app/src/main/java/com/x8bit/bitwarden/data/platform/manager/ReviewPromptManager.kt

```diff
@@ -0,0 +1,27 @@
+package com.x8bit.bitwarden.data.platform.manager
+
+/**
+ * Responsible for managing whether or not the app review prompt should be shown.
+ */
+interface ReviewPromptManager {
+    /**
+     * Increments the add cipher item action count for the active user.
+     */
+    fun incrementAddCipherActionCount()
```

Comment:
⛏️ 💭 What do you think about renaming these `increment` functions to something like `registerAddCipherAction`? I only ask because we're not always incrementing the counter. Or mentioning the exception case in kdocs would be enough if you prefer the current name.

Reply:
like that suggestion a lot.

Reply:
also need to change rest a bit, we are moving away from saving it per user base.